libraries/lib-datahandler/src/main/thrift/components.thrift

```diff
@@ -238,6 +242,8 @@ struct Component {
     33: optional set<string> releaseIds,
 
     35: optional set<string> mainLicenseIds,        //Aggregate of release main licenses
+    36: optional set<string> mainLicenseNames,
+    37: optional set<License> mainLicenses,
```

Comment:
is this ever used apart from being set in ComponentPortlet.java:479? I can't seem to find any usages